composeApp/src/commonMain/kotlin/org/ooni/probe/data/repositories/PreferenceRepository.kt

```diff
@@ -104,6 +139,34 @@ class PreferenceRepository(
         }
     }
 
+    suspend fun getProxyString(): String {
```

Comment:
Now we can integrate this proxy string into the Engine: https://github.com/ooni/probe-multiplatform/blob/06a6cbff292e2ef7f1c5b74ba787092839b329c5/composeApp/src/commonMain/kotlin/org/ooni/probe/domain/GetEnginePreferences.kt#L25